docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,172 @@
+# Creating the vSphere Windows VM Golden Image
+
+This guide describes the thought process of creating a Windows virtual machine base image (the Golden Image) in vSphere.
+
+
+## 1. Select a compatible Windows Server version
+
+Currently the Windows Machine Config Operator (WMCO) stable version [2.0.0](REF?) only supports Windows Server version 1909 (build 18363.\*) Semi-Annual Channel, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351), required to get the operating system feature that allows setting the [hybrid OVN Kubernetes networking with a custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere)
```

Comment:
`(REF?)` doesnt point to anything

Reply:
I would refrain from mentioning any version, it should be assumed that this is talking about the current version

Reply:
Agree, left that as an open forum to review. Thanks

Reply:
If this was left to an open forum, the conversation should not be marked as resolved.

Not sure what `REF` implies. I say we just call out the Windows version here.